metacat-common-server/src/main/java/com/netflix/metacat/common/server/properties/HiveProperties.java

```diff
@@ -44,6 +44,8 @@
 
         @NonNull
         private Partition partition = new Partition();
+        private int fetchSize = 2500;
+        private int batchSize = 2500;
 
         /**
```

Comment:
Shouldn't this go to the hive connector? These properties are hive connector specific and shouldn't leak into the server config.

Reply:
There is already a TODO for this class. For now I have added it here. We can move it later. We need to have a system default setting and also have at the catalog level.